logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserRepository.kt

```diff
@@ -708,6 +709,10 @@ internal class UserDataSource internal constructor(
         clientDAO.isMLSCapable(userId.toDao(), clientId.value)
     }
 
+    override suspend fun getContactsSize(): Either<StorageFailure, Int> = wrapStorageRequest {
+        userDAO.countUsers()
+    }
```

Comment:
Just to be sure... I am not sure what a "contact" is. `userDAO.countUsers()` is returning a count of _known_ users. These users not necessarily have connection requests, etc.

Edit: I scrolled down and saw the query is explicitly querying accepted connections, not only "known users".

Maybe `getEstablishedConnectionsCount` instead of `getContactsSize` ?